includes/admin/views/setup-wizard/main.php

```diff
@@ -2,8 +2,10 @@
 /**
  * Setup Wizard main view
  *
- * @since 4.4.4
- * @version 4.8.0
+ * @package LifterLMS/Views/Admin/SetupWizard
+ *
+ * @since   4.4.4
+ * @version [version]
```

Comment:
```suggestion
 * @since 4.4.4
 * @since 4.8.0 Unknown.
 * @since [version] Some comments on what you did here.
 * @version [version]
```

We use to keep a bit of changelog on *templatish*